subsys/mpsl/mpsl_fem.c

```diff
@@ -289,6 +289,7 @@ static int fem_nrf21540_gpio_configure(void)
 	}
 #endif
 	(void)err;
+	(void)inactive_pin_configure;
```

Comment:
Please don't do this for functions.

Either surround the function in appropriate ifdef.
Or use `if(IS_ENABLED(CONFIG_FEATURE))` to avoid this problem.

Reply:
no appropriate CONFIG_FEATURE for function. Instead of adding DT_NODE_HAS_PROP to ifdef, I make it as an inline function.

Reply:
```
if (IS_ENABLED(DT_NODE_HAS_PROP(DT_NODELABEL(nrf_radio_fem), mode_gpios)) {
	err = inactive_pin_configure(
		DT_GPIO_PIN(DT_NODELABEL(nrf_radio_fem), mode_gpios),
		DT_GPIO_LABEL(DT_NODELABEL(nrf_radio_fem), mode_gpios),
		DT_GPIO_FLAGS(DT_NODELABEL(nrf_radio_fem), mode_gpios));
	if (err) {
		return err;
	}
}
```
I was thinking something like this, I think that should work. Inlining a function just to avoid this problem isn't the best solution IMO.